src/components/StyleGuide/ColorCard.tsx

```diff
@@ -590,6 +591,55 @@ export const cssVars = {
       textColor: colorContrastData.ui.gray.xLightWarm.textColor,
     },
   ],
+  highlighter: [
+    {
+      name: "ui.highlighter.red",
+      showColorDataTable: false,
+      textColor: "ui.black",
```

Comment:
Can this value be the default so it doesn't have to be added for every object?

Reply:
The text color assignment needs to be set based on the background color for the card, so there is no single color value that work for all instances of the `ColorCard`. What I can do is set a default color value and then only add a value when the default needs to be overridden.

Reply:
Ha! I already have the text defaulting to `ui.white`, so I will remove all of the value in all the instance where I am setting the text to white.